components/automate-ui/src/app/modules/desktop/dashboard/dashboard.component.scss

```diff
@@ -1,2 +1,172 @@
 @import "~styles/variables";
+@import "@carbon/layout/scss/layout";
+@import "@carbon/grid/scss/grid";
 
+:host {
+  display: block;
+  height: 100%;
+}
+
+.dashboard-grid {
+  @extend .bx--grid;
```

Comment:
extend rules 👍 

for anyone looking for more info: https://css-tricks.com/the-extend-concept/

Reply:
Agreed. This pattern is pretty much required for this view anyway. The [first iterations](https://github.com/chef/automate/blob/sc/desktop-dash-layout/components/automate-ui/src/app/modules/desktop/dashboard/dashboard.component.html#L6) I had used the grid classNames directly in the HTML. It quickly became a bit unwieldy to have chunks of code like `[ngClass]="{'bx--col-lg-8': !insightVisible, 'bx--col-lg-16': insightVisible}"` all over the place